torchtitan/models/llama3/__init__.py

```diff
@@ -39,6 +39,15 @@
         use_flex_attn=True,
         attn_mask_type="block_causal",
     ),
+    "debugmodel_varlen_attn": TransformerModelArgs(
+        dim=256,
+        n_layers=6,
+        n_heads=16,
+        vocab_size=2048,
+        rope_theta=500000,
+        use_varlen_attn=True,
+        attn_mask_type="varlen_attn",
```

Comment:
From what I understand, varlen attention is a type of attention and use it as a mask field sound a little bit confusing